src/fmu/dataio/datastructure/configuration/global_configuration.py

```diff
@@ -83,9 +83,25 @@ class Access(BaseModel):
     @classmethod
     def _classification_mirros_accesslevel(cls, values: Dict) -> Dict:
         if isinstance(ssdl := values.get("ssdl", {}), dict):
-            values["classification"] = ssdl.get("access_level")
+            values["classification"] = ssdl.get("access_level", "restricted")
             if values["classification"] == "asset":
                 values["classification"] = "restricted"
+
+        return values
+
+    @model_validator(mode="before")
+    @classmethod
+    def _modify_access_level_if_deprecated_value(cls, values: Dict) -> Dict:
+        if (
+            isinstance(ssdl := values.get("ssdl", {}), dict)
+            and ssdl.get("access_level") == "asset"
+        ):
+            warnings.warn(
+                "Using 'asset' as 'access_level' is deprecated,"
+                "set to 'restricted' instead.",
+                FutureWarning,
+            )
+            ssdl["access_level"] = "restricted"
```

Comment:
Note these will be adressed by @janbjorge in a separate PR. There were some implementation in that was not supported in our Access model. Will be rebased later.